### ISSUE TITLE
A11y: Search input filter contains image which ALT text label is missing.

### ISSUE THREAD

**Issue by kimclow** [Automattic/wp-calypso#78369]
### Quick summary

On the site, the Search button contains elements whose ALT text is missing which violate the WCAG accessibility text standards.

Screenshot: https://d.pr/i/KMyBv6

I have specified that it's most users as being affected, as I don't know what percentage of users have a site search on their site. 

### Steps to reproduce

1. Add a Gutenberg search block onto a site
2. Save the draft or page 
3. Load the page on the front end 
4. Inspect the element and review A11y feedback 
5. You'll see the missing alt tag 

### What you expected to happen

I expect our block to meet WCAG accessibility standards. 

### What actually happened

Our block fails the WCAG accessibility test. 

### Impact

Most (> 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Self-hosted
